tidy3d/components/simulation.py

```diff
@@ -436,6 +437,21 @@ def _warn_monitor_simulation_frequency_range(cls, val, values):
                 )
         return val
 
+    @pydantic.validator("monitors", always=True)
+    def n2f_monitors_with_symmetry(cls, val, values):
+        """Error if a server-side near-to-far monitor is used with symmetry"""
+        monitors = val
+        symmetry = values.get("symmetry")
+        if not any(symmetry):
+            return val
+        for monitor in monitors:
+            if isinstance(monitor, AbstractNear2FarMonitor):
+                raise SetupError(
+                    "Server-side near-to-far calculation is not available with symmetry. "
+                    "Use the local near-to-far available through the ``RadiationVectors`` class."
```

Comment:
We were discussing this with @tylerflex recently and it seems what we should be doing is using simple ' ' (as in `'RadiationVectors'` in error messages (while still using ` ``RadiationVectors `` ` in docstrings). We'll be trying to do that in the future (also @weiliangjin2021 @dbochkov-flexcompute )

Reply:
Yea, agree. it renders better in std out and in notebooks too.

Reply:
Docstrings also means descriptions of pydantic Fields by the way. These get scraped to build the class docstring.